packages/contracts-bedrock/interfaces/safe/ITimelockGuard.sol

```diff
@@ -16,6 +16,8 @@ interface ITimelockGuard {
         uint256 executionTime;
         TransactionState state;
         ExecTransactionParams params;
+        bytes32 txHash;
```

Comment:
I would move the `txHash` to the first position in the struct, since it's its identifier.